metadata-ingestion/setup.py

```diff
@@ -302,7 +301,7 @@ def get_long_description():
     # https://github.com/elastic/elasticsearch-py/issues/1639#issuecomment-883587433
     "elasticsearch": {"elasticsearch==7.13.4"},
     "feast": {
-        "feast~=0.29.0",
+        "feast>=0.30.2",
```

Comment:
Let's keep this as a ~= dep. it's a bit more controlled

Reply:
Will do in separate PR so I can fix CI first